extensions/mutiny/runtime/src/main/java/io/quarkus/mutiny/runtime/MutinyInfrastructure.java

```diff
@@ -11,4 +15,23 @@
     public void configureMutinyInfrastructure(ExecutorService exec) {
         Infrastructure.setDefaultExecutor(exec);
     }
+
+    public void configureDroppedExceptionHandler() {
+        Infrastructure.setDroppedExceptionHandler(new Consumer<Throwable>() {
+            @Override
+            public void accept(Throwable throwable) {
+                Logger.getLogger(MutinyInfrastructure.class).error("Mutiny had to drop the following exception", throwable);
+            }
+        });
+    }
+
+    public void configureThreadBlockingChecker() {
+        Infrastructure.setCanCallerThreadBeBlockedSupplier(new BooleanSupplier() {
+            @Override
+            public boolean getAsBoolean() {
+                String threadName = Thread.currentThread().getName();
+                return !threadName.contains("vertx-eventloop-thread-");
```

Comment:
Can you add a comment about why we use that strategy (avoid depending on Vert.x explicitly, also avoid to select worker threads).

Reply:
Yep